pkgs/applications/misc/glava/default.nix

```diff
@@ -43,12 +33,12 @@ let
 in
   stdenv.mkDerivation rec {
     name = "glava-${version}";
+    version = "1.5.1";
 
-    src = fetchFromGitHub {
-      owner = "wacossusca34";
-      repo = "glava";
+    src = fetchgit {
```

Comment:
Why did you change this to fetchgit?

Reply:
I didn't seem to get the glad submodule when I used fetchFromGithub, since the current revision in the repository works now.